### ISSUE TITLE
Replace kanso

### ISSUE THREAD

**Issue by garethbowen** [medic/medic-webapp#3019]
kanso is dead. We need to move off it.

### Package management

We’ve already moved completely to using npm (via bower…). The remaining packages are only needed for kanso deployment.

### Ddoc building

There are grunt libraries for compiling and deploying couchapps, eg: https://github.com/jo/grunt-couch
We may need to do some custom code to bundle up api and sentinel for pushing to the market, just like we have custom code (via grunt tasks) to compile js, css, and html assets.

**Comment by garethbowen** [comment]
Related: #3017 #3018

**Comment by nice-snek** [comment]
hi frand @abbyad, U is coot

Please close or schedule before the end of this sprint. See [triaging old issues](https://github.com/medic/medic-docs/blob/master/md/dev/workflow.md#triaging-old-issues).

Nice Snek

**Comment by abbyad** [comment]
Marking this as scheduled, and will leave @garethbowen and @sglangevin to pull this into the appropriate iteration.

**Comment by mandric** [comment]
WIP https://github.com/medic/medic-docs/blob/ddoc-refactor/rfcs/ddoc-refactor.md

**Comment by mandric** [comment]
(note to self: sorry force pushes with issues linked can be noisy)

**Comment by mandric** [comment]
Status update: PR is open here https://github.com/medic/medic-api/pull/149 for medic-api, decided to wait on merge for that to also create the webapp part of that to officially deprecate /add in medic-webapp and return 50x or friendly error message and potentially clean up few other things.

**Comment by garethbowen** [comment]
Scheduling for 3.0.0 as it goes nicely with the Horti changes.

**Comment by garethbowen** [comment]
Leave the above PR to be merged once `master` is 3.0.0 and document under breaking changes that medic-transport and smssync will no longer work.

**Comment by ngaruko** [comment]
I assume there is no particular test required for this, apart from checking that the app still works as expected, particular pieces that depended on or used `kanso`. I am going to close this and if an issue comes up due to this work, we will open a separate issue. Any objection @garethbowen @abbyad ?
